.circleci/config.yml

```diff
@@ -147,7 +147,7 @@ jobs:
       - *attach_workspace
       - run:
           name: "deploy"
-          command: "sudo npm i -g serverless && sls deploy -s mosaic-prod --CLIENTEMAIL $CLIENT_EMAIL --PRIVATEKEY $PRIVATE_KEY"
+          command: "sudo npm i -g serverless && sls deploy -s mosaic-prod --CLIENT_EMAIL $CLIENT_EMAIL --PRIVATE_KEY $PRIVATE_KEY --TEMPLATE_DOCUMENT_ID $TEMPLATE_DOCUMENT_ID --TITLE $TITLE"
```

Comment:
Thing for the future, we can install `serverless` as a dev dependency of the project itself, which means you won't need to do `sudo npm i -g` here. I'd suggest we pick that up this week.

Main benefit is that we can lock the serverless version in our package.json (rather than this which will always install the latest version, which might introduce a breaking change in the future)

Reply:
I'll create ticket for this.